.github/workflows/reuse-int-tests.yml

```diff
@@ -68,6 +68,10 @@ on:
         type: string
         required: false
         default: "main"
+      pr_head_sha:
+        type: string
+        required: true
+        default: ''
```

Comment:
Since `pr_head_sha` is always required we can leave out default. Will work as intended

```suggestion
```